### ISSUE TITLE
Changes in table column order break old calculations setups

### ISSUE THREAD

**Issue by haasad** [LCA-ActivityBrowser/activity-browser#118]
Old order above with same activity added with new order:
![image](https://user-images.githubusercontent.com/11636405/37609472-7d85e0aa-2b9d-11e8-9619-4925782a04e5.png)
```python
Traceback (most recent call last):
  File "/home/adrian/Git/github/AB/activity-browser/activity_browser/app/ui/tabs/lca_results.py", line 70, in calculate
    self.mlca = MLCA(name)
  File "/home/adrian/Git/github/AB/activity-browser/activity_browser/app/bwutils/multilca.py", line 48, in __init__
    self.lca.redo_lci(func_unit)
  File "/home/adrian/miniconda3/envs/abe/lib/python3.6/site-packages/bw2calc/lca.py", line 497, in redo_lci
    self.build_demand_array(demand)
  File "/home/adrian/miniconda3/envs/abe/lib/python3.6/site-packages/bw2calc/lca.py", line 137, in build_demand_array
    self.demand_array[self.product_dict[key]] = demand[key]
ValueError: could not convert string to float: 'kilogram'
```

**Comment by haasad** [comment]
Activity, Location and Database are also empty when new activities are added:
![image](https://user-images.githubusercontent.com/11636405/37650168-6d10e75a-2c34-11e8-9107-7cf7636c80f2.png)
This disappears when the CS is reloaded, columns then also arranged in the correct order:
![image](https://user-images.githubusercontent.com/11636405/37650230-9fb3c696-2c34-11e8-90f0-181e78319ec7.png)

**Comment by haasad** [comment]
Fixed in #119
